Microsoft.Azure.Cosmos.Samples/Usage/Queries/Program.cs

```diff
@@ -224,6 +226,61 @@ private static async Task QueryWithSqlParameters(Container container)
         }
         // </QueryWithSqlParameters>
 
+        // <QueryWithContinuationTokens>
+        private static async Task QueryWithContinuationTokens(Container container)
+        {
+            // Query using two properties within each item. WHERE Id == "" AND Address.City == ""
+            // notice here how we are doing an equality comparison on the string value of City
+
+            QueryDefinition query = new QueryDefinition("SELECT * FROM c");
+            string continuation = null;
+
+            List<Family> results = new List<Family>();
+            using (FeedIterator<Family> resultSetIterator = container.GetItemQueryIterator<Family>(
+                query,
+                requestOptions: new QueryRequestOptions()
+                {
+                    MaxItemCount = 1
+                }))
+            {
+                // Execute first page of query
+                FeedResponse<Family> response = await resultSetIterator.ReadNextAsync();
+
+                results.AddRange(response);
+                if (response.Diagnostics != null)
```

Comment:
I am not aware of a case where this is null, you can just do:
```suggestion
                Console.WriteLine($"\nQueryWithSqlParameters Diagnostics: {response.Diagnostics}");
```

Reply:
I believe there are scenarios in query where it can be null. I'm not sure if that is true any more but older SDK versions it was possible.